announcer.py

```diff
@@ -36,7 +36,7 @@ def announce(self):
                 if self.slacker.channel_exists(config.announce_channel):
                     self.sb.say(config.announce_channel, m)
                 else:
-                    self.ds.warning("Attempted to announce in {}, but channel does not exist.".format(config.announce_channel))
+                    self.ds.logger.warning("Attempted to announce in %s, but channel does not exist.", config.announce_channel)
```

Comment:
@h3h the % method of formatting is deprecated (https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations). Is there any reason not to stick with the format style?